src/server/rest/v1/service/UserService.ts

```diff
@@ -1316,6 +1316,7 @@ export default class UserService {
         issuer: filteredRequest.Issuer,
         siteIDs: (filteredRequest.SiteID ? filteredRequest.SiteID.split('|') : null),
         userIDs: (filteredRequest.UserID ? filteredRequest.UserID.split('|') : null),
+        tagID: (filteredRequest.TagID ? filteredRequest.TagID : null),
```

Comment:
Does the filter allow several tags ID? 
If yes, you have to use an array.

Reply:
No, at the moment the filter only supports on TagID.

Reply:
Thanks for mentioning, it does need to support several tag IDs! Working on the change now.